docs/tools/vdb_table/data/azureai.json

```diff
@@ -143,7 +143,7 @@
     "comment": "Limited by batch size. However, the maximum size of a document is 16 MB. There is no max document limit per index."
   },
   "vector_dims": {
-    "value": 2048,
+    "value": 3072,
     "unlimited": false,
     "source_url": "https://learn.microsoft.com/azure/search/vector-search-how-to-generate-embeddings",
```

Comment:
The docs URL linked here still says 2048 dimensions is the limit. Can that be updated first or maybe link to somewhere that actually shows 3072?

Reply:
It is updated now. Merging